yb-voyager/cmd/analyzeSchema.go

```diff
@@ -784,6 +785,14 @@ func analyzeSchema() {
 		utils.ErrExit("failed to write report to %q: %s", reportPath, err)
 	}
 	fmt.Printf("-- find schema analysis report at: %s\n", reportPath)
+
+	callhome.InitJSON(exportDir)
+	payload := callhome.GetPayload()
+	issues, _ := json.Marshal(reportStruct.Issues)
```

Comment:
Do not ignore errors. Log an ERROR message and return from the callhome functionality.

Same comment applies to other similar occurrences.

Reply:
I hadn't cached the error as there was no scenario in which an error would occur over here- I will still add this functionality for consistency in code 👍🏽

Reply:
Please log the error and return. There can be many funny ways in which code can fail.